src/axom/sidre/docs/sphinx/query_data.rst

```diff
@@ -0,0 +1,94 @@
+.. ## Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _dataconcepts-label:
+
+===========================
+Datastore Queries
+===========================
+
+Sidre provides a couple of methods that can be called to query certain
```

Comment:
Can we provide an example on how to print the information to the screen on this page?

Reply:
Follow-up: Would it be helpful to provide a print function that takes in the filled conduit node:

```
   Datastore* ds = ...;

  conduit::Node n;
  ds->getBufferInfo(n);
  ds->printBufferInfo(n);  // Guess doesn't really need to be a class member function but helpful maybe for finding it
```

or a parameter on the get to just print):

```
   Datastore* ds = ...;

  conduit::Node n;
  ds->getBufferInfo(n, printToScreen=true);
```

Reply:
Conduit Node provides ways to print the contents of a Node. I think that is sufficient. Agree?

Reply:
I personally don't know how to do that off the top of my head and would have to look it up. Could you add that to this example?

Reply:
added